libs/initScripts/computeRSet_with_HCs.js

```diff
@@ -0,0 +1,346 @@
+const axios = require('axios')
+const fs = require('fs')
+
+const AudiusLibs = require('../src/index')
+const Util = require('../src/utils')
+const CreatorNode = require('../src/services/creatorNode')
+
+const CONTENT_NODE_TYPE = 'content-node'
+
+// PROD
+// const ETH_PROVIDER_URL = 'https://eth.audius.co'
+// const DISCOVERY_NODE_ENDPOINT = 'https://discoveryprovider.audius.co'
+// const USER_METADATA_ENDPOINT = 'https://usermetadata.audius.co/'
+// const ETH_REGISTRY_ADDRESS = '0xd976d3b4f4e22a238c1A736b6612D22f17b6f64C'
+// const ETH_TOKEN_ADDRESS = '0x18aAA7115705e8be94bfFEBDE57Af9BFc265B998'
+// const ETH_OWNER_WALLET = '0xC7310a03e930DD659E15305ed7e1F5Df0F0426C5'
+
+// STAGING
+// const ETH_PROVIDER_URL = 'https://eth.staging.audius.co'
+// const DISCOVERY_NODE_ENDPOINT = 'https://discoveryprovider.staging.audius.co'
+// const ETH_REGISTRY_ADDRESS = '0xe39b1cA04fc06c416c4eaBd188Cb1330b8FED781'
+// const ETH_TOKEN_ADDRESS = '0x74f24429ec3708fc21381e017194A5711E93B751'
+// const ETH_OWNER_WALLET = '0xcccc7428648c4AdC0ae262D3547584dDAE25c465'
+
+// NOTE: Migrate URSM first via `node setup.js run user-replica-set-manager up`
+
+// LOCAL
+const ethContractsConfig = require('../eth-contracts/config.json')
+const dataContractsConfig = require('../data-contracts/config.json')
+const ETH_PROVIDER_ENDPOINT = 'http://localhost:8546'
+const DISCOVERY_NODE_ENDPOINT = 'http://localhost:5000'
+const DATA_CONTRACTS_PROVIDER_ENDPOINT = 'http://localhost:8545'
+const USER_METADATA_ENDPOINT = 'http://cn-um_creator-node_1:4099'
```

Comment:
likely need this in staging/prod too right? same with several of these configs - let's make a note of that bc it's actually super nice to have them above

Reply:
will add more for staging/prod today!